cscs-checks/tools/profiling_and_debugging/ddt.py

```diff
@@ -120,12 +117,11 @@ def __init__(self, lang, extension):
             '--trace-at _jacobi-cuda-kernel.cu:111,residue'
         ]
 
-        self.flags += ['-DUSE_MPI']
-        self.flags += ['-D_CSCS_ITMAX=5']
+        self.build_system.cflags = ['-DUSE_MPI', '-D_CSCS_ITMAX=5']
```

Comment:
These are `cppflags` (i.e., preprocessor flags). Here, you will only set the `CFLAGS` in the `Makefile` invocation.